packages/cli/src/util/config.ts

```diff
@@ -58,7 +58,7 @@ const CONFIG_FILE_SCHEMA: JSONSchemaType<FoundryConfig> = {
           oneOf: [
             {
               properties: {
-                "type": { enum: ["git-describe"], type: "string" },
+                "type": { const: "git-describe", type: "string" },
```

Comment:
Is this because we are using oneOf so if we introduce a new "type" we will just add a new object?

Reply:
Yes, I believe so otherwise we wouldn't use oneOf at all if we wanted to express everything under one object but then you wouldn't get the stricter validation on which properties belong to which type in the union and everything would have to be nullable